Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -188,7 +188,14 @@ extension BrowserViewController: WKNavigationDelegate {
             decisionHandler(.cancel)
             return
         }
+        
         let isPrivateBrowsing = PrivateBrowsingManager.shared.isPrivateBrowsing
+        
```

Comment:
Can you add a comment referencing the ticket we are fixing?

I think we should use this old ticket for it 
https://github.com/brave/brave-ios/issues/511